src/test/ui/feature-gates/feature-gate-asm_unwind.rs

```diff
@@ -0,0 +1,8 @@
+#![feature(asm)]
```

Comment:
@Amanieu Do you think it's alright if I put `// only-x86_64` here? (since most of the feature gate tests are `only-x86_64`)

Reply:
That's fine.